vipps-recurring-api-faq.md

```diff
@@ -63,6 +65,14 @@ the user.
 No, the user needs to contact the Merchant which can then cancel or modify the
 agreement as they see fit.
 
+## What happens to charges if the user cancels the agreement?
+All charges in a `PENDING` or `DUE` state will be cancelled if the agreement is stopped.
+
+## What happens to pending charges if the user deletes the payment card?
+We are going to update the card handling logic on our side and won't let a user
```

Comment:
please remove this, we don't need to lay out our roadmap
you can write:
`The user is responsible for keeping their payment sources up-to-date. They will receive a push notification if a payment fails which informs them to update their payment source. We are looking to improve user experience here in the future`

Reply:
An open roadmap can be useful, and this has already been communicated externally - but I can of course remove it.

Reply:
An open roadmap can be useful, and this has already been communicated externally - but I can of course remove it.